9.2/root/usr/bin/run-postgresql

```diff
@@ -16,5 +16,10 @@ fi
 
 set_passwords
 
+if [ -f ${CONTAINER_SCRIPTS_PATH}/post-init ]; then
```

Comment:
@pkajaba yes, in `README` there can be the basic information and an example. In case of mysql we also include empty `post-init` file, so people find the location easily and then it makes sense to me also include some information (or link) in this file as well.